src/codegen/ruby/java_types.rb

```diff
@@ -43,7 +44,15 @@
   'string' => 'String',
   'timestamps object' => 'ActivityTimestamps',
   'user object' => 'User',
-  'user id object' => 'UserId'
+  'user id object' => 'UserId',
+  'partial guild object' => 'Guild',
+  'partial channel object' => 'Channel',
+  'account object' => 'IntegrationAccount',
+  'array of integrations' => 'ImmutableList<Integration>',
+  'array of audit log change objects' => 'ImmutableList<AuditLogChange>',
+  'optional audit entry info' => 'OptionalAuditEntryInfo',
+  'audit log event' => 'Integer',
+  'array of audit log entry objects' => 'ImmutableList<AuditLogEntry>'
```

Comment:
alphabetical order by key please. makes it easier to see if something already has been mapped

Reply:
My bad, will fix that.